zyte_common_items/base.py

```diff
@@ -118,3 +120,52 @@ def _apply_field_types_to_sub_fields(cls, item: Optional[Dict]):
             item.update({key: cls.from_list(item.get(key, [])) for key, cls in (from_list or {}).items()})
 
         return item
+
+
+@export
+@attrs.define(slots=True)
+class AdditionalProperty:
+    name: str
+    value: str
+
+
+@export
+@attrs.define(slots=True, kw_only=True)
```

Comment:
[I did it](https://github.com/zytedata/zyte-common-items/pull/2/commits/9f3992b4f31132409710827a261a8c2127a9e2bb), and it passed regular tests, but not mypy, so I [reverted](https://github.com/zytedata/zyte-common-items/pull/2/commits/a3511d2431cc418c543613bde3c42bc859e9079f). First I tried a few things with `functools.wraps`, but nothing I tried worked for mypy. I am sure it _can_ be done, but I do not know the right way to do it. Please, feel free to do it yourself if you feel motivated :slightly_smiling_face:

Reply:
Let's tackle it outside this PR to prevent it from being a blocker. I've created a separate issue in https://github.com/zytedata/zyte-common-items/issues/5.

Reply:
We can also drop the `export` decorator :)